install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -129,18 +129,50 @@ It is recommended to deploy both groups to avoid potential impacts on
 performance in I/O and volume creation. Both of these are defined in the
 inventory hosts file.
 
-The definition of the clusters is done by including the relevant names in the
-`[OSEv3:children]` group, creating similarly named groups, and then populating
-the groups with the node information. The clusters can then be configured
-through a variety of variables in the `[OSEv3:vars]` group. `glusterfs`
-variables begin with `openshift_storage_glusterfs_` and `glusterfs_registry`
-variables begin with `openshift_storage_glusterfs_registry_`. A few other
-variables, such as `openshift_hosted_registry_storage_kind`, interact with the
-GlusterFS clusters.
-
-To prevent {gluster} pods from upgrading after an outage leading to a cluster
-with different {gluster} versions, it is recommended to specify the image name
-and version tags for all containerized components. The relevant variables are:
+To define the storage clusters, include the relevant names in the
+`[OSEv3:children]` group, creating similarly named groups. Then populate
+the groups with the node information.
+
+After the groups are created and populated, you can then configure the clusters
+using a variety of variables in the `[OSEv3:vars]` group. The variables interact
+with the GlusterFS clusters.
+
+- `glusterfs` variables begin with `openshift_storage_glusterfs_`.
+- `glusterfs_registry` variables begin with `openshift_storage_glusterfs_registry_`.
+
+[[install-advanced-installer]]
+==== Host variables
+
+Each host in the `glusterfs` and `glusterfs_registry` groups must have the
+`glusterfs_devices` variable defined. This variable defines the list of block
+devices that will be managed as part of the GlusterFS cluster. You must have at
+least one device, which must be bare, with no partitions or LVM PVs.
+
+You can also define the following variables for each host. If they are defined,
+these variables further control the host configuration as a GlusterFS node:
+
+- `glusterfs_cluster`: The ID of the cluster this node should belong to.
+- `glusterfs_hostname`: A hostname (or IP address) to be used for internal
+  GlusterFS communication.
+- `glusterfs_ip`: The IP address that the pods will use to communicate with
+the GlusterFS node.
+- `glusterfs_zone`: A zone number for the node. Within the cluster, zones
+determine how to distribute the bricks of GlusterFS volumes.
+
+[[install-advanced-installer]]
+==== Role variables
+
+To control the integration of a GlusterFS cluster into a new or existing {product-title}
+cluster, you can also define a number of role variables. Each role variable also
+has a corresponding variable to optionally configure a separate GlusterFS cluster
+for use as storage for an integrated Docker registry.
+
+[[install-advanced-installer]]
+==== Image name and Version tag variables
```

Comment:
Please confirm the capitalization here.

Reply:
using as nouns, so made "version" lower case, so "Image name and version tag variables" per our sentence style cap.